lib/LDAP.php

```diff
@@ -183,7 +183,7 @@ public function nextEntry($link, $result) {
 	}
 
 	/**
-	 * @param LDAP $link
+	 * @param LDAP|LDAP[] $link
```

Comment:
the same as above. In addition PHPDoc for the Interface method should be updated accordingly.

Reply:
fixed